src/core/css/ErrorHandler.scss

```diff
@@ -7,4 +7,8 @@
   padding: 10px;
   margin: 0;
   margin-bottom: 10px;
+
+  > li {
```

Comment:
If that's the case it's fine, my two _actually compelling_ reasons for using classes are:

1. it's convention we've been following thus far
2. it means we target things very specifically and never rely on nesting for styles

Obviously the refutation of those points is easy: 1. who cares 2. `<li>` elements are always inside `<ul>`

So wfm; in the future I won't write out classes on `<li>`s either and it'll likely be nicer 😄